content/md/onboarder/1-set-up-onboarder.md

```diff
@@ -21,7 +21,7 @@ The [technical prerequisites](https://docs.akeneo.com/latest/onboarder/prerequis
 ![Onboarder list](../img/onboarder_list_waiting_setup.png)
 
 1. First, from the menu, go to `Project console / Projects` and select your project.
-2. Go to the Onboarder instance tab, and open the instance with the status `Waiting for setup`, by clicking on the instance line.
+2. Go to the Onboarder instance tab, and open the instance with the status `Waiting for activation setup`, by clicking on the instance line.
```

Comment:
to add to the other setup page (Flex, Serenity, Shared catalogs)

Reply:
On the setup we need to add the confirmation modal step:
`1. On the confirmation modal, click "Confirm the request"
`